internal/bminventory/inventory.go

```diff
@@ -279,6 +279,11 @@ func (b *bareMetalInventory) formatIgnitionFile(cluster *common.Cluster, params
 		"SkipCertVerification": strconv.FormatBool(b.SkipCertVerification),
 		"AgentTimeoutStartSec": strconv.FormatInt(int64(b.AgentTimeoutStart.Seconds()), 10),
 	}
+	if safeForLogs {
+		for _, key := range []string{"userSshKey", "PullSecretToken", "PULL_SECRET", "RH_ROOT_CA"} {
```

Comment:
Does the public part of a CA or the public part of an ssh key need to be considered secret?

Reply:
well I guess strictly not, but I also thought what is the benefit of printing out a huge CA/ssh key?
I am happy to take it out if you think it is useful to have in.